addons/mail/static/src/new/public/discuss_public.xml

```diff
@@ -2,9 +2,12 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.discuss_public" owl="1">
-    <div>
-        DISCUSS PUBLIC
-    </div>
+    <div class="o_DiscussPublicView d-flex flex-column h-100 w-100 position-absolute" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
Use the current convention for naming classnames, which is kebab-case: `o-mail-discuss-public`